README.md

```diff
@@ -1,16 +1,204 @@
 # Documentation Guidelines
+The official https://appwrite.io/docs documentation source code.
 
 ## 🚀 Contributing
+1. Clone the repository
+2. Update, add, or fix current docs
+3. Once content is ready, raise a PR
 
 ## Before Proposing Additions
+Documentation is an integral part of Appwrite and follows the same philosophy of quick to get started, easy to grow. Before proposing additions to Appwrite's documentation, think about where these additions fit best.
+- The **Getting Started** section helps a developer create their first Appwrite project and make their first requests so they can begin exploring Appwrite. Avoid adding unnecessary information to this section so a developer's first-impressions remains quick and smooth.
+- The **Guides** section helps a developer get comfortable with individual Appwrite services. Guides are structured to walk a developer through features of an Appwrite service. The topics should be ordered by level of complexity, where complex topics are positioned later into a guide.
+- The **REST API** section helps a developer understand details of individual endpoints. Additions to REST API documentation should remain focused on the API endpoint itself. Avoid adding information about related endpoints and services that are not integral to undestanding documented endpoint. Code example changes should be submitted to the [SDK Generator repo](https://github.com/appwrite/sdk-generator).
```

Comment:
We could also mention that endpoint names or descriptions are to be updated in appwrite/appwrite.